R/qenv-get_env.R

```diff
@@ -0,0 +1,32 @@
+#' Extract Environment Included in `qenv`
+#'
+#' The extraction of environment included in `qenv@env` allows to list object names included in `qenv@env` slot.
```

Comment:
- "Access"? An "extract" feels too complicated to me.
- (...) allows to **e.g.** list object names

Reply:
Access is fine :)
5d32b50c44b845d84975a1a808f182a3f9eab67a